serverless/serverless-getting-started.adoc

```diff
@@ -13,23 +13,32 @@ include::modules/technology-preview.adoc[leveloffset=+1]
 
 [id="how-it-works_{context}"]
 == How {ServerlessProductName} works
-Developers on {ServerlessProductName} can use the provided Kubernetes-native APIs, as well as familiar languages and frameworks, to deploy applications and container workloads.
-For information about installing {ServerlessProductName}, see xref:../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[Installing OpenShift Serverless].
 
-{ServerlessProductName} on {product-title} enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
+Developers on {ServerlessProductName} can use the provided Kubernetes native APIs, as well as familiar languages and frameworks, to deploy applications and container workloads.
+
+{ServerlessProductName} on {product-title} enables stateless serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
 
 {ServerlessProductName} is based on the open source Knative project, which provides portability and consistency across hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
 
-[id="knative-eventing_{context}"]
-== Knative Eventing
-A developer preview version of Knative Eventing is available for use with {ServerlessProductName}. However, this is not included in the {ServerlessOperatorName} and is not currently supported as part of this Technology Preview. For more information about Knative Eventing, including installation instructions and samples, see the link:https://openshift-knative.github.io/docs/docs/index.html[Knative Eventing on {product-title}] documentation.
+// [id="serverless-ha_{context}"]
+// == High availability in {ServerlessProductName}
```

Comment:
Is this all commented out? Do you plan to remove the slashes?

Reply:
Yes but not until GA, I would usually wait to merge the PR at release but 1. I have a bunch of them open and want to merge as many as possible, and 2. if I comment it out I can just go ahead and merge the changes for other parts of the docs and uncomment this later for GA :)